.travis.yml

```diff
@@ -2,22 +2,53 @@ sudo: true
 dist: bionic
 language: generic
 
+# We get this encrypted variable by:
+# 1) Setup the cache in google cloud storage as per https://docs.bazel.build/versions/master/remote-caching.html
+# 2) getting secure key as a `.json` from https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating_a_service_account 
+# 3) Encrypting the `.json` via https://docs.travis-ci.com/user/encrypting-files/ 
+
+before_install:
+  - ls
+  # Decrypt the key for uploading to the remote bazel cache if we're not in 
+  # a pull request (Travis does not allow encrypted variables in public PR's)
+  - >
+    if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then 
+      openssl aes-256-cbc -K $encrypted_b311edfc2b7f_key -iv $encrypted_b311edfc2b7f_iv -in bazel-remote-cache-key.json.enc -out bazel-remote-cache-key.json -d
+    fi
+
+env:
+  - PATH_TO_REMOTE_BAZEL_CACHE_KEY="../bazel-remote-cache-key.json"
```

Comment:
Where is this relative path from? The root of the repository, meaning this is at the same level as the `Software` repo in CI's filesystem?

Reply:
Relative to the root for bazel, which is `src`

Reply:
Thoughts on some comment we should add here?

Reply:
ahh I see now. Maybe just "This path is relative to the bazel root, which is our `Software/src` directory`?

Reply:
added this (tweaked in case we ever change folder structure :stuck_out_tongue: )
```
  # This path is relative to the bazel root directory (ie. where `WORKSPACE`)
  # is located
```

Reply:
:+1: